.eslintrc.yml

```diff
@@ -0,0 +1,60 @@
+extends:
+- airbnb-base
+- next
+
+env:
+  browser: true
+  es2021: true
+  es6: true
+  jest: true
+
+globals:
+  google: readonly
+  module: readonly
+  process: readonly
+
+parserOptions:
+  ecmaVersion: 8
+  sourceType: module
+  ecmaFeatures:
+    jsx: true
+
+rules:
+  arrow-body-style: off
+  arrow-parens: off
+  func-names: warn
+  func-style: off
+  max-len: warn
+  no-bitwise: off
+  no-cond-assign:
+  - warn
+  - except-parens
+  no-confusing-arrow: off
+  no-console:
+  - warn
+  - allow:
+    - warn
+    - error
+  no-mixed-operators: off
+  no-nested-ternary: off
+  no-plusplus: warn
+  no-restricted-syntax: off
+  no-shadow: off
+  no-undef: warn
+  no-unused-vars: warn
+  no-use-before-define:
+  - warn
+  - functions: false
+    classes: false
+  object-curly-newline: off
+  prefer-destructuring: off
```

Comment:
I disabled [this rule](https://eslint.org/docs/rules/prefer-destructuring) even though I really like it because when I was demoing Next and wrote `{ id } =`, people were confused. If you think they can handle required destructuring, I'm all for turning it back on.

Reply:
Change this to a warning; I want people to learn how to use this style but not be blocked by not using it.